src/components/NavBreadcrumbs.js

```diff
@@ -40,6 +39,11 @@ export default function NavBreadcrumbs(props) {
     color: activeLinkColor,
     opacity: FULL_OPAQUE,
   }
+  const linkPropsNotActive = {
+    paddingBottom: '1px',
+    borderBottomStyle: 'solid',
+    borderBottomWidth: '0.1px',
+  }
```

Comment:
If the default breadcrumb style needs to be modified can  ```linkProps``` be updated instead of adding a new ```linkPropsNotActive``` class?  Also you can remove ```textDecoration: 'none'``` to underline the breadcrumb text instead of creating a bottom border.

https://www.w3schools.com/cssref/pr_text_text-decoration.asp

If you need to remove the underline for the last crumb in the list of crumbs you can try using ```last-child```

https://www.w3schools.com/cssref/sel_last-child.asp

Keep in mind working with style classes in javascript is a bit different than working in pure css.  For examples of ```last-child``` look at accordionSection.js or src/pages/TagCreator/AddOrgForm/styles.js

Once you take care of the styling directly, then you shouldn't need to deal with ```notActiveStyle``` logic

Reply:
If I give text-decoration:underline the styling on website looks not good since the underline is touching the words on top. If thats fine then I can remove the paddingBottom. Attaching a screenshot for your reference.
![Screenshot (17)](https://user-images.githubusercontent.com/6700926/130730733-ab35b052-4f74-4f2e-86d2-4ef093f08b8a.png)

Reply:
Also for this one I don't understand what you are saying
If you need to remove the underline for the last crumb in the list of crumbs you can try using last-child

https://www.w3schools.com/cssref/sel_last-child.asp

Because I am asked to underline the links that was not active.But if you are using last-child it is underlining  active page link.

Can you guide me how to solve this?